content/en/getting_started/application/monitors.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Getting Started with Monitors
+kind: documentation
+further_reading:
+- link: "https://www.datadoghq.com/blog/monitoring-101-alerting/"
+  tag: "Blog"
+  text: "Monitoring 101: Alerting on what matters"
+- link: "/monitors/monitor_types/metric/"
+  tag: "Documentation"
+  text: "Metric Monitors"
+- link: "/monitors/notifications/"
+  tag: "Documentation"
+  text: "Monitor Notifications"
+---
+
+## Overview
+
+A [metric monitor][1] provides alerts and notifications if a specific metric is above or below a certain threshold. This page provides instructions for setting up a metric monitor to alert on low disk space.
+
+## Prerequisites
+
+Before getting started, you need a [Datadog account][2] linked to a host with the [Datadog Agent][3] installed. To verify, check your [Infrastructure List][4] in Datadog.
+
+## Setup
+
+To create a [metric monitor][5] in Datadog, use the main navigation: *Monitors --> New Monitor --> Metric*.
+
+### Choose the detection method
+
+When you create a metric monitor, **Threshold Alert** is automatically selected as the detection method. A threshold alert compares metric values to a static threshold. The goal for this monitor is to alert on a static threshold, so no change is necessary.
+
+### Define the metric
+
+To get an alert on low disk space, use the `system.disk.in_use` metric from the [Disk integration][6] and average the metric over `host` and `device`:
+
+{{< img src="getting_started/application/metric_query.png" alt="alert setup" responsive="true" >}}
+
+After this is set, the monitor automatically updates to a `Multi Alert` that triggers a separate alert for each `host`, `device` reporting your metric.
+
+### Set alert conditions
+
+According to the [Disk integration documentation][6], `system.disk.in_use` is *the amount of disk space in use as a fraction of the total*. So, when this metric is reporting a value of `0.7`, the device is 70% full.
+
+To alert on low disk space, the monitor should trigger when the metric is `above` the threshold. The threshold values are based on your preference. For this metric, values between `0` and `1` are appropriate:
+
+{{< img src="getting_started/application/alert_thresholds.png" alt="alert setup" responsive="true" >}}
+
+For this example, the other settings in this section are left on the defaults. For more details, see the [Metric Monitors][7] documentation.
+
+### Say what's happening
+
+Before a monitor can be saved, it must have a title and message.
+
+#### Title
+
+The title must be unique for each monitor. Since this is a multi alert monitor, names are available for each group (`host` and `device`) with message template variables:
```

Comment:
Technically the groups for this monitor are "`host`, `device`" so I'd rephrase to something like:
`names are available for each group element (`host` and `device`)`